packages/rum/src/browser/requestIdleCallBack.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { requestIdleCallback } from './requestIdleCallback'
+
+describe('requestIdleCallback', () => {
+  let requestAnimationFrameSpy: jasmine.Spy
+  let cancelAnimationFrameSpy: jasmine.Spy
+  let callback: jasmine.Spy
+  const originalRequestIdleCallback = window.requestIdleCallback
+  const originalCancelIdleCallback = window.cancelIdleCallback
+
+  beforeEach(() => {
+    requestAnimationFrameSpy = spyOn(window, 'requestAnimationFrame').and.callFake((cb) => {
+      cb(0)
+      return 123
+    })
+    cancelAnimationFrameSpy = spyOn(window, 'cancelAnimationFrame')
+    callback = jasmine.createSpy('callback')
+  })
+
+  afterEach(() => {
+    window.requestIdleCallback = originalRequestIdleCallback
+    window.cancelIdleCallback = originalCancelIdleCallback
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏No need to do this since you use `spyOn`.